db/migrate/20220205090438_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
Please add `author_id` as a foreign key instead of `user_id` and reference it to the User model.